system/Session/Session.php

```diff
@@ -317,35 +379,35 @@ public function set($data, $value = null)
 	//--------------------------------------------------------------------
 
 	/**
-	 * Get any user data that has been set in the session.
+	 * Get user data that has been set in the session.
 	 *
+	 * If the property exists as "normal", returns it.
+	 * Otherwise, returns an array of any temp or flash data values with the
+	 * property key.
+	 * 
 	 * Replaces the legacy method $session->userdata();
 	 *
-	 * @param null $key
-	 *
-	 * @return array|null
+	 * @param  $key	Identifier of the session property to retrieve
+	 * @return array|null	The property value(s)
 	 */
 	public function get($key = null)
 	{
 		if (isset($key))
 		{
 			return isset($_SESSION[$key]) ? $_SESSION[$key] : null;
-		}
-		elseif (empty($_SESSION))
+		} elseif (empty($_SESSION))
```

Comment:
`elseif` should be on a new line. 

We also need to decide on a version to use, either `elseif` or `else if`. I've seen it both ways in the changes, and I know I'm probably guilty of this. I used to always use the two word version, but think I've started to use the single word more often due to some of the nagging style checks over at Sensio's style checker. :)